rpcs3/util/asm.hpp

```diff
@@ -37,45 +38,45 @@ namespace utils
 	template <typename F, typename R = std::invoke_result_t<F>>
 	inline auto tx_start(F op)
 	{
-#if defined(ARCH_X64)
+#if defined(ARCH_X64) && defined(__RTM__)
```

Comment:
The changes in this file are a little sketchy. I see some blocks switched around and I wonder if the functions are supported on non-msvc compilers. I'm guessing the RTM path is currently disabled?

Reply:
I will revert my changes.